organisaatio-ui/src/components/Sivut/Ryhmat/Ryhmat.tsx

```diff
@@ -4,45 +4,43 @@ import TyypitJaRyhmatKehys from '../TyypitJaRyhmatKehys/TyypitJaRyhmatKehys';
 import { useContext } from 'react';
 import { KoodistoContext, LanguageContext } from '../../../contexts/contexts';
 import Button from '@opetushallitus/virkailija-ui-components/Button';
-import Input from '@opetushallitus/virkailija-ui-components/Input';
-import { Icon } from '@iconify/react';
-import searchIcon from '@iconify/icons-fa-solid/search';
-import Select from '@opetushallitus/virkailija-ui-components/Select';
 import { useEffect } from 'react';
 import Axios from 'axios';
 import { useState } from 'react';
 import { Ryhma } from '../../../types/types';
 import NormaaliTaulukko from '../../Taulukot/NormaaliTaulukko';
 import Spin from '@opetushallitus/virkailija-ui-components/Spin';
+import { Column } from 'react-table';
 
 const urlPrefix = process.env.NODE_ENV === 'development' ? '/api' : '/organisaatio';
 
 const Ryhmat: React.FC = (props) => {
```

Comment:
what is "props" and what it is used for?

Reply:
will remove props